src/components/Articles/Articles.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from 'react'
+import { getArticles } from '../../../APIs'
+import { Article } from '../Article/Article'
+import './Articles.css'
+
+export const Articles = ({ articles, setArticles }) => {
+    useEffect(() => {
+        getArticles()
+            .then((articles) => {
+                setArticles(articles)
```

Comment:
I would add some loading state here so the user can see something when it takes a while for them to load